sdcm/mgmt.py

```diff
@@ -314,8 +314,11 @@ def create_backup_task(self, param_dict):
             raise ScyllaManagerError("Unknown failure for sctool '{}' command".format(command))
 
         if res.stderr:
-            LOGGER.debug("Encountered an error on '{}' command response".format(command))
-            raise ScyllaManagerError(res.stderr)
+            if res.stderr.count('unable to resolve host'):
```

Comment:
please add comment why we are ignoring those errors

Reply:
added